src/redux/epics/getStoreLocations.epic.js

```diff
@@ -1,22 +1,40 @@
+import Rx from "rxjs";
+
 import {
   STORE_LOCATION_ACTIONS,
-  sendStoresByProductIDs
-} from "../actions/storeLocation.actions"
+  sendStoresByProductIDs,
+  setUserLocation,
+} from "../actions/storeLocation.actions";
 
-import { LCBO_API_KEY, LCBO_BASE_URL } from "../constants"
+import { LCBO_API_KEY, LCBO_BASE_URL } from "../constants";
 
-const productIDs = ids => ids.join(",")
+const productIDs = (ids=[]) => ids.join(",");
 
 export const getStoreLocations = (action$, _, { ajax }) =>
   action$
     .ofType(STORE_LOCATION_ACTIONS.GET_STORES_BY_PRODUCT_IDS)
-    .mergeMap(action =>
-      ajax(
-        `${LCBO_BASE_URL}stores?products=${productIDs(
-          action.payload
-        )}&lat=43.653&lon=-79.383&access_key=${LCBO_API_KEY}`
-      )
-    )
-    .map(({ response }) => {
-      return sendStoresByProductIDs(response.result)
+    .mergeMap(action => {
+      return ajax(
+        `${LCBO_BASE_URL}stores?products=${productIDs(action.payload.ids)}&lat=${action.payload.latLng
+          .lat}&lon=${action.payload.latLng.lng}&access_key=${LCBO_API_KEY}`,
+      );
     })
+    .map(({ response }) => {
+      return sendStoresByProductIDs(response.result);
+    });
+
+function getPreciseLocation() {
+  return new Promise(function(resolve, reject) {
+    navigator.geolocation.getCurrentPosition(function(position) {
+      resolve({ lat: position.coords.latitude, lng: position.coords.longitude });
```

Comment:
I think we should handle reject here as well (even if we just console.log an error message for us to use while we're developing)